deploy/docs/Security_Best_Practices.md

```diff
@@ -39,6 +39,52 @@ requests over the network.
 The log metadata enrichment service needs to be able to talk to the Sumo Logic backend receiver endpoints. It also needs
 to be able to access the Kubernetes API Server to obtain metadata.
 
+Example Kubernetes Network Policies restricting all but the necessary traffic for the logs pipeline:
+
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: fluent-bit
+spec:
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/name: fluent-bit
+  policyTypes:
+  - Ingress
+  - Egress
```

Comment:
```suggestion
  policyTypes:
  - Egress
```

Reply:
This is intentional, adding `Ingress` here and then not providing a configuration for it means "reject all inbound connections.

Reply:
I feel like if someone is actually following this guide, they should already know how Network Policies work.

Reply:
People might also learn stuff from our docs :)

Reply:
I realized we should actually allow trafic on Fluent-Bit's metrics port, so I added it here along with a comment.